apps/frontend/vite.config.mts

```diff
@@ -0,0 +1,46 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig, splitVendorChunkPlugin } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [splitVendorChunkPlugin(), react()],
+  server: {
+    open: true,
+    host: true,
+    port: 3000,
+    proxy: {
+      ...[
+        '/graphql',
+        '/downloads',
+        '/download',
+        '/uploads',
+        '/files',
+        '/preview',
+      ].reduce((acc, curr) => {
+        acc[curr] = {
+          target: 'http://localhost:4000',
```

Comment:
The same goes here. Maybe it is better to read this from environment variable first and have fallback to `http://localhost:4000` if that's not set.